api-reference/10 UI Components/dxLookup/1 Configuration/dropDownOptions.md

```diff
@@ -7,7 +7,8 @@ type: dxPopover_Options
 Configures the drop-down field.
 
 ---
-Refer to the [Popover configuration](/api-reference/10%20UI%20Components/dxPopover '/Documentation/ApiReference/UI_Components/dxPopover/') for information on properties that you can specify in this object.
+
+Based on the [usePopover](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#usePopover)'s value, refer to the [Popover](/api-reference/10%20UI%20Components/dxPopover '/Documentation/ApiReference/UI_Components/dxPopover/') or [Popup](/Documentation/ApiReference/UI_Components/dxPopup/) configuration for properties that you can specify in the **dropDownOptions** object.
```

Comment:
The type is strictly dxPopover_Options. Although the dependency between usePopover and dropDownOptions seems logical, I'm not sure that it is indeed programmed.